.github/workflows/cpaas-sync-drivers.yml

```diff
@@ -73,14 +100,89 @@ jobs:
               gsutil -m cp -n -I "gs://${{ inputs.drivers-bucket }}/${{ matrix.platform }}/$(basename "${driver_version_dir}")/"
           done
 
+      # x86 support packages will not be uploaded to GCP.
+      # This is to prevent mixing it up with the upstream built packages.
```

Comment:
Consider finding a way of skipping steps if matrix.platform != 'x86_64'

Reply:
I've put some thought into this one and I could try to move all the steps to a separate workflow, then make only that workflow run on the non x86 case, but I would also need to either:
- Upload the support-packages as an artifact for the follow up workflow to download it.
- Directly use the images from a separate workflow for extracting the support-packages directly.

Both seem to me as extra complexity just to avoid putting `if matrix.platform != 'x86_64'` in 3 steps. I think I'd prefer to keep it as is, but if you feel strongly about it, I can change it.

Reply:
I don't feel strongly about it.